packages/tx/src/eip1559Transaction.ts

```diff
@@ -196,12 +205,9 @@ export default class FeeMarketEIP1559Transaction extends BaseTransaction<FeeMark
    * The up front amount that an account must have for this transaction to be valid
    * @param baseFee The base fee of the block (will be set to 0 if not provided)
    */
-  getUpfrontCost(baseFee?: BN): BN {
-    if (!baseFee) {
-      baseFee = new BN(0)
-    }
-    const inclusionFeePerGas = BN.min(this.maxPriorityFeePerGas, this.maxFeePerGas.sub(baseFee!))
-    const gasPrice = inclusionFeePerGas.add(baseFee!)
+  getUpfrontCost(baseFee: BN = new BN(0)): BN {
```

Comment:
there are no changes here, just inlining the default baseFee in the method argument (https://github.com/ethereumjs/ethereumjs-monorepo/pull/1148#discussion_r637437606)